metadata.json

```diff
@@ -43,8 +43,9 @@
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "26",
-        "27"
+        "31",
```

Comment:
Fedora 31 is now EOL

Reply:
damn it! I checked wikipedia and it said it's still supported :D I will remove it

Reply:
Went EOL in the past few days. May even have been yesterday.